### DISCUSSION TITLE
Task list filter AND OR NOT

### DISCUSSION THREAD

**Discussion by tscahill** [zim-desktop-wiki/zim-desktop-wiki#2365] [Ideas]
This idea stems from trying to filter what is displayed on the task list. I'm not sure if this issue extends beyond my present linux build (Kubuntu. I've been running it since December). My experience is that in the task list, I can only select one tag, because the others are immediately filtered from the tag view. Reality check: Is this how it is working across other OS and desktops?

Having 1) just come to understand that the Search function can do keywords and operators, and 2) seeing that those work as expected when the current search query is 4) automatically pasted into the subsequent "Find" dialog of pages 5) loaded by clicking on the search dialog results; could the Task list filter bar be modified to be a bit more boolean? I'm asking for the ability to filter the list more granularly. 

Of course, right now I'm experimenting with making meta-tags across a range of other tags to manually get the Task List result I'm looking for across a range of other tags. That works, but it involves parsing through existing todo items to manually add meta-tags. I have a lot of to-dos. I can continue to do it that way if what I'm asking about wouldn't be of benefit to many other users. 

Of course, I'd welcome any suggestions on alternatives ways to accomplish this.

**Comment by brucewestfall** [comment]
Not totally sure what you're looking for, but in my case i use about 4-8 tags for each job.  Then after i have too many to remember, i can narrow things down selecting tags one at a time.  I do drawings for indoor and outdoor basketball, tennis, pickleball, etc.  Residential and then mostly schools.  @indoor @outdoor @84x50 @pickle @4square, and about 20 more.
Everything gets either indoor or outdoor.  I dont bother with a @basketball tag since it is the vast majority and won't really help much.  If you have other words you want to be tags, just do a find and replace and change all the "volleyball" into "@volley"
After a couple rounds of that you will have a nice tag selection without much trouble.  I always list my tags together, but they can be in the middle of a paragraph and still be quickly found.

Just sharing a few thoughts.  Maybe it can help a little.
